### PR TITLE
Add 'Paste new UUID' command

### PR DESCRIPTION
**Pull request by simon-pearson**
Hi, another command I use which I thought I'd share.

### CODE REVIEW

**Comment by hluk** [comment]
Cool, this is handy. Can you please rebase (conflict with your previous merged pull request)?

**Comment by simon-pearson-lmk** [comment]
Ah yes, of course - all done 👍

File: Global/paste-new-uuid.ini [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        var d1 = Math.random()*0xffffffff|0;
+        var d2 = Math.random()*0xffffffff|0;
+        var d3 = Math.random()*0xffffffff|0;
+        return lut[d0&0xff]+lut[d0>>8&0xff]+lut[d0>>16&0xff]+lut[d0>>24&0xff]+'-'+
+            lut[d1&0xff]+lut[d1>>8&0xff]+'-'+lut[d1>>16&0x0f|0x40]+lut[d1>>24&0xff]+'-'+
+            lut[d2&0x3f|0x80]+lut[d2>>8&0xff]+'-'+lut[d2>>16&0xff]+lut[d2>>24&0xff]+
+            lut[d3&0xff]+lut[d3>>8&0xff]+lut[d3>>16&0xff]+lut[d3>>24&0xff];
+    }
+    
+    var lut = [];
```
Comment (hluk) [inline Global/paste-new-uuid.ini:20]: How about moving this also into `generateUuid()`?

**Comment by hluk** [comment]
Thanks!

**Pull request merged**